en/installation/macos.md

```diff
@@ -106,4 +108,31 @@ brew install chromedriver
 brew install imagemagick
 ```
 
-> Now you're ready to go get Consul [installed](../local_installation.html)!!
```

Comment:
Instead of copying the instructions here, is it possible to fix the link instead? In other places we use `[installed](local_installation.md)`, so maybe that's the right place here as well :thinking:.